lib/storage/bucket.js

```diff
@@ -65,7 +65,7 @@ var RESUMABLE_THRESHOLD = 5000000;
  * @constructor
  * @alias module:storage/bucket
  *
- * @throws if a bucket name isn't provided.
+ * @throws {Error} if a bucket name isn't provided.
```

Comment:
If you call *deleting code until it passes then seeing what you deleted to find the issue* debugging.